modin/config/pubsub.py

```diff
@@ -70,7 +70,7 @@ class ExactStr(str):
     ),
     int: TypeDescriptor(
         decode=lambda value: int(value.strip()),
-        normalize=int,
+        normalize=int,  # type: ignore
```

Comment:
could you explain the reasoning for the ignore here?

Reply:
Yes, this was the error that I was getting:
```
modin/config/pubsub.py:73:19: error: Argument "normalize" to "TypeDescriptor" has incompatible type "Type[int]"; expected "Callable[[object], object]"  [arg-type]
```

I think `mypy` is assuming `int` to be a type instead of using the hint to make use of it as a function to cast something to an int. I tried to make it more explicit by writing something like this:
```python
normalize=lambda value: int(value)
```
but then I got hit with these warnings:
```
modin/config/pubsub.py:73:33: error: No overload variant of "int" matches argument type "object"  [call-overload]
modin/config/pubsub.py:73:33: note: Possible overload variants:
modin/config/pubsub.py:73:33: note:     def int(cls, Union[str, bytes, array[Any], mmap, _CData, PickleBuffer, SupportsInt, SupportsIndex, SupportsTrunc] = ...) -> int
modin/config/pubsub.py:73:33: note:     def int(cls, Union[str, bytes], base: SupportsIndex) -> int
```